WordPress/Classes/ViewRelated/Notifications/Views/NotificationDetailUserView.swift

```diff
@@ -0,0 +1,174 @@
+import SwiftUI
+import DesignSystem
+
+struct NotificationDetailUserView: View {
+	typealias FollowConfig = NotificationDetailUserView.FollowActionConfiguration
+
+    let config: Configuration
+
+    init(avatarURL: URL?, username: String?, blog: String?, onUserClicked: @escaping () -> Void) {
+        self.config = Configuration(avatarURL: avatarURL, username: username, blog: blog, onUserClicked: onUserClicked)
+    }
+
+    init(
+        avatarURL: URL?,
+        username: String?,
+        blog: String?,
+        isFollowed: Bool,
```

Comment:
Wouldn't it make sense to make `isFollowed` an optional argument with default value of nil instead of defining 2 inits?

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/23178/commits/1d7ed27a784ace46ca53cd5d4dafe4007d1bf9a2 🔨